app/static/settings_style.css

```diff
@@ -0,0 +1,143 @@
+/* Settings page */
+
+.sub-title {
+    font-size: 1rem;
+    padding-top: 0.2rem;
+}
+
+.settings {
+  color: #222831;
+  display: flex;
+  font-size: 0.9rem;
+}
+
+#settings-left {
+  padding: 1.5rem 1.5rem;
```

Comment:
Prefer working with em wherever possible.

Reply:
@aviadamar told me to use rem, it's his fault!
סתם! תיקנתי :)